anaSbitReadout.py

```diff
@@ -0,0 +1,376 @@
+#!/bin/env python
+"""
+    Script to analyze sbitreadout data
+    By: Caterina Aruta (caterina.aruta@cern.ch) and Francesco Ivone (francesco.ivone@cern.ch)
+    """
+if __name__ == '__main__':
+    import ROOT as r
+    import numpy as np
+    import root_numpy as rp
+    import glob
+    import sys
+    import os
+    import time
+    from optparse import OptionParser
+    from subprocess import call
+    from array import array
+    from gempython.utils.nesteddict import nesteddict as ndict
+    from gempython.gemplotting.mapping.chamberInfo import chamber_iEta2VFATPos
+    from gempython.gemplotting.mapping.chamberInfo import chamber_vfatPos2iEtaiPhi as vfat_to_etaphi
+    from gempython.gemplotting.utils.anautilities import make3x8Canvas, saveSummaryByiEta, getMapping
+    from gempython.gemplotting.utils.anaoptions import parser
+    import pkg_resources
+
+    parser.set_defaults(outfilename="sbitReadOut.root")
+    (options, args) = parser.parse_args()
+    path = options.filename
+    size = ((options.GEBtype).lower())
+
+    if not os.path.isdir(path):
+        raise OSError(2, "No such file or directory", path)
+    else:
+        pass
+
+    if size not in ('long', 'short'):
+        raise AssertionError("Invalid value of GEBtype")
+    else:
+        pass
+
+    filename = options.filename+"/sbitReadOut"
+    outfilename = options.outfilename
+    print("Analyzing: '%s'" % path)
```

Comment:
All print statements should appear like line 41 so that they are `python3` compatible.  You have some print statements that are not `python3` compatible.

Reply:
Done